### PR TITLE
fix: php drivers namespace

### PR DESCRIPTION
**Pull request by clementvtrd**
update PHPDriver and StaticPHPDriver namespace to Doctrine/Persistence.

Related to issue #1793

### CODE REVIEW

File: config/orm.xml [hunk 1]
```diff
@@ -33,8 +33,8 @@
         <parameter key="doctrine.orm.metadata.annotation.class">Doctrine\ORM\Mapping\Driver\AnnotationDriver</parameter>
         <parameter key="doctrine.orm.metadata.xml.class">Doctrine\ORM\Mapping\Driver\SimplifiedXmlDriver</parameter>
         <parameter key="doctrine.orm.metadata.yml.class">Doctrine\ORM\Mapping\Driver\SimplifiedYamlDriver</parameter>
-        <parameter key="doctrine.orm.metadata.php.class">Doctrine\ORM\Mapping\Driver\PHPDriver</parameter>
-        <parameter key="doctrine.orm.metadata.staticphp.class">Doctrine\ORM\Mapping\Driver\StaticPHPDriver</parameter>
+        <parameter key="doctrine.orm.metadata.php.class">Doctrine\ORM\Persistence\Driver\PHPDriver</parameter>
```
Comment (dmaicher) [inline config/orm.xml]: Shouldn't this be

```suggestion
        <parameter key="doctrine.orm.metadata.php.class">Doctrine\Persistence\Mapping\Driver\PHPDriver</parameter>
```

?

Comment (alexpozzi) [reply config/orm.xml]: It seems correct according to the upgrade documentation https://github.com/doctrine/orm/blob/3.1.x/UPGRADE.md#metadata-drivers

**Review by alexpozzi — APPROVED** [review]

File: config/orm.xml (hunk 1 above)
Comment (clementvtrd) [reply config/orm.xml]: Actually you're right I made a typo nice catch! I did it well for StaticPHPDriver not for PHPDriver ✅

**Review by stof — APPROVED** [review]

**Comment by ostrolucky** [comment]
Dupe of https://github.com/doctrine/DoctrineBundle/pull/1755

**Pull request closed without merging**